main-site/lib/data.ts

```diff
@@ -96,3 +96,35 @@ export const FaqSectionData: FaqData[] = [
       'To have a great time at HackBeanpot 2022, you should prepare your workspace with your computer, scrap paper for ideation, any hardware you may want for your hack and your best space cadet energy to learn and create new things! Stay tuned for an Astronaut Training Manual in your inbox sent out closer to the event with more details.'
   }
 ];
+
+export const testimonialSectionData: TestimonialData[] = [
+  {
+    id: 0,
+    author: 'Karyna Yen',
+    year: 'Northeastern 2025',
```

Comment:
maybe say c/o 2025 or '25 here instead?

Reply:
I think '25 looks better, what do you think? @deanframe 

<img width="432" alt="Screen Shot 2022-10-21 at 12 56 13 PM" src="https://user-images.githubusercontent.com/103906123/197249165-88ffab43-7bae-44ab-a1da-4a63e7420b3a.png">

Reply:
^^ i agree '25 looks better

Reply:
+1 for '25